test-network-function/generic/suite.go

```diff
@@ -164,7 +164,8 @@ var _ = ginkgo.Describe(testsKey, func() {
 		log.Info(containersUnderTest)
 
 		ginkgo.Context("Container does not have additional packages installed", func() {
-			if os.Getenv("FSDIFF") == "1" {
+			// use this boolean to turn off tests that require OS packages
+			if isMinikube() {
```

Comment:
This is the inverse of what's needed. We only want to run the test when the env var is false, this will only run it when the var is true.
Please make sure to check that the test runs when the env var is not set.

Reply:
Switch to:
```
if !isMinikube() {
...
```

Reply:
i think my last merge/rebase screwed the changes i did, will correct. thanks for point out.